src/mixins/embed-button.js

```diff
@@ -108,16 +108,24 @@ export default {
     async postReadEvent() {
       if (this.isRead) return;
       this.isRead = true;
-      await apiPostLikeButtonReadEvent(
-        this.id,
-        {
-          referrer: this.referrer,
-          isCookieSupport: this.hasCookieSupport,
-          documentReferrer: this.documentReferrer,
-          sessionID: this.sessionId,
-          type: this.buttonType,
-        },
-      );
+      try {
+        await apiPostLikeButtonReadEvent(
+          this.id,
+          {
+            referrer: this.referrer,
+            isCookieSupport: this.hasCookieSupport,
+            documentReferrer: this.documentReferrer,
+            sessionID: this.sessionId,
+            type: this.buttonType,
+          },
+        );
+      } catch (err) {
+        const errMsg = err.message || err.toString();
+        if (!errMsg.includes('timeout')
+          && !errMsg.includes('Network Error')) {
+          console.error(err);
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/17264716/89051038-3961a380-d386-11ea-9fe8-af6859de83fa.png)

Reply:
It turns out our current lint setting does not like prettier version
![image](https://user-images.githubusercontent.com/6198816/89063934-a03d8780-d39b-11ea-9221-82431dc030c4.png)

Reply:
Will this solve?
```js
if (
  !errMsg.includes('timeout')
  && !errMsg.includes('Network Error')
) {
   console.error(err);
}
```